test/plugin/test_out_google_cloud.rb

```diff
@@ -224,7 +231,17 @@ def test_one_log_private_key
     verify_log_entries(1, COMPUTE_PARAMS)
   end
 
+  def test_one_log_custom_metadata
```

Comment:
ack - although this test is norsly a superset of the earlier test, I think they're still both valuable.